menas/src/main/resources/application.properties.template

```diff
@@ -17,6 +17,9 @@
 # unlimited set the property to -1.
 spring.servlet.multipart.max-file-size=16MB
 
+# Set the timeout a single connection request can wait for a controller to return a value (in milliseconds)
+spring.mvc.async.request-timeout=600000
```

Comment:
A default of 10 minutes sounds like a lot. 1 minute (60000)?

Reply:
Yeah, I suppose the reasonable default value is 1 minute.